prqlc/prqlc-parser/src/parser/pr/mod.rs

```diff
@@ -0,0 +1,21 @@
+//this exports everything within as if it was at the root.
+//eg: PR::expr::Expr becomes PR::Expr
+pub use expr::*;
+pub use ident::*;
+pub use ops::*;
+pub use stmt::*;
+pub use types::*;
```

Comment:
This is pretty aggressive exporting. No need to delay merge, but was this the case before? If not, would be good to trim it down at some point...

Reply:
OK it was like this prior! Closing

Reply:
yep, another place for export optimization. Saving that for the next 'pass'